_test/test_multifit_herbert/test_multifit_1.m

```diff
@@ -324,5 +331,45 @@
             assertEqualToTolWithSave (this, ww_fcellarr_f, 'tol', tol)
             assertEqualToTolWithSave (this, pp_fcellarr, 'tol', tol)
         end
+
+        function test_multifit_simple_against_parallel(obj)
+            clOb = set_temporary_config_options('hpc_config','parallel_multifit',false);
+            vcryst0 = [...
+                0.0467    1.1561    0.1316
+                -1.2565    2.5545   -1.1844
+                0.0655   -0.1970    1.3143];
+
+            rlu_expected =[...
+                0    -1     0
+                1     2     0
+                0    -1     1];
+            nv = 3;
+
+            vcryst0_3 = repmat(vcryst0',3,1);     % Treble the number of vectors, as will compute three components of deviations
+
+            pars = [...
+                5.0000    5.0000    5.0000...
+                90.0000   90.0000   90.0000...
+                -0.0514    0.0611    0.0362];
+
+            kk = multifit (vcryst0_3, zeros(3*nv,1), 0.01*ones(3*nv,1));
+
+            kk = kk.set_fun (@reciprocal_space_deviation, {pars,rlu_expected});
+            kk = kk.set_options ('list', 0);
+            kk = kk.set_options ('fit',[1e-4,50,-1e-6]);
+
+            [distance,fitpar] = kk.fit();
+            assertEqualWithSave(obj,distance,'tol',1.e-4);
+            assertEqualWithSave(obj,fitpar,'tol',1.e-3);
+            clear clOb;
+
+            skipTest('Re #1724 This fit fails and should be fixed as part of the ticket #1724')
```

Comment:
this suggests that the fix is going to be enabling parallel multifit. But the fix in this PR is to disable parallel multifit. At the least this message needs modifying

Reply:
parallel multifit is disabled in `crystal_alignment`. This test shows that parallel multifit fails for a task, similar to `crystal_aligmnent` but not actually it. You may see that no `crystal_alignment` is invoked here.